common/dbus/platform/asio_libdbus/dbus.cpp

```diff
@@ -417,7 +417,7 @@ ExpectedStringPair ExtractValueFromDBusMessage(DBusMessage *message) {
 		dbus_error_free(&dbus_error);
 		return expected::unexpected(err);
 	}
-	return std::pair<string, string> {string(value1), string(value1)};
+	return StringPair {string(value1), string(value2)};
```

Comment:
Did we not comment on this in the original PR? :thinking:

Reply:
Yes, moved the fix into a separate commit and then moved the fix commit to this fix PR. :grin: